### PR TITLE
(#1048) Refactor GzipInput and GzipOutput tests to be more readable

### PR DESCRIPTION
**Pull request by Umbrah**
This if for #1048 
I replaced data bytes with simple numbers

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1118](https://github.com/yegor256/cactoos/issues/1118) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1118](https://github.com/yegor256/cactoos/issues/1118) is assigned to @ilyakharlamov[/z](https://www.0crat.com/u/ilyakharlamov), here is [why](https://www.0crat.com/footprint/C63314D6Z/e9f1fa95-bf62-4bda-ad63-a70408e9f464); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by ilyakharlamov** [comment]
@llorllale it's OK to merge that, I've double checked the numbers

File: src/test/java/org/cactoos/io/GzipInputTest.java [hunk 1]
```diff
@@ -43,13 +43,11 @@
...
         final byte[] bytes = {
             (byte) GZIPInputStream.GZIP_MAGIC,
-            // @checkstyle MagicNumberCheck (1 line)
-            (byte) (GZIPInputStream.GZIP_MAGIC >> 8), (byte) 0x08,
-            (byte) 0x00, (byte) 0x00, (byte) 0x00, (byte) 0x00, (byte) 0x00,
-            (byte) 0x00, (byte) 0x00, (byte) 0xF3, (byte) 0x48, (byte) 0xCD,
-            (byte) 0xC9, (byte) 0xC9, (byte) 0x57, (byte) 0x04, (byte) 0x00,
-            (byte) 0x56, (byte) 0xCC, (byte) 0x2A, (byte) 0x9D, (byte) 0x06,
-            (byte) 0x00, (byte) 0x00, (byte) 0x00,
+            -117, 8, 0, 0, 0,
```
Comment (llorllale) [inline src/test/java/org/cactoos/io/GzipInputTest.java]: @Umbrah in the case of `GzipInput`, the problem I see is that these are a bunch of magic numbers that the reader needs to either trust that they represent "Hello!", or put in some effort to verify it themselves on paper or somewhere else.

The readability of this test can be improved by using the Java SDK to serialize "Hello!" to an array of gzipped bytes. Then, these same bytes are read back with `GzipInput` and the contents have to be the same "Hello!" string. The benefits here are that a) there are no magic numbers, and b) we set up a nice BUILD-OPERATE-CHECK pattern where there is a clear correlation between the BUILD and the CHECK.

File: src/test/java/org/cactoos/io/GzipOutputTest.java [hunk 2]
```diff
@@ -53,13 +53,11 @@
...
         final byte[] bytes = {
             (byte) GZIPInputStream.GZIP_MAGIC,
-            // @checkstyle MagicNumberCheck (1 line)
-            (byte) (GZIPInputStream.GZIP_MAGIC >> 8), (byte) 0x08,
-            (byte) 0x00, (byte) 0x00, (byte) 0x00, (byte) 0x00, (byte) 0x00,
-            (byte) 0x00, (byte) 0x00, (byte) 0xF3, (byte) 0x48, (byte) 0xCD,
-            (byte) 0xC9, (byte) 0xC9, (byte) 0x57, (byte) 0x04, (byte) 0x00,
-            (byte) 0x56, (byte) 0xCC, (byte) 0x2A, (byte) 0x9D, (byte) 0x06,
-            (byte) 0x00, (byte) 0x00, (byte) 0x00,
+            -117, 8, 0, 0, 0,
```
Comment (llorllale) [inline src/test/java/org/cactoos/io/GzipOutputTest.java]: @Umbrah same general principles apply here as I have outlined above

**Comment by llorllale** [comment]
@Umbrah see my comments

**Comment by Umbrah** [comment]
@llorllale Fixed according to your comment, please check

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @ilyakharlamov[/z](https://www.0crat.com/u/ilyakharlamov), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1118](https://github.com/yegor256/cactoos/issues/1118) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by sereshqua** [comment]
@ilyakharlamov you haven't found any issues during code review, which means quality is bad here.

please confirm that next time you would find at least 3 issue during the code review

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Pull request merged**